services/datamanager/data/fake_robot_with_disabled_collector.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Added this since updating in `testCfg` would properly get passed to `Update()` but then get converted back to its original value with the call to `TransformAttributeMapToStruct`

In addition to a massive refactor (clean up and extract builtin.go into multiple files), want to clean up tests to have an object-oriented way to easily update all parts of the config.